### ISSUE TITLE
Issue with flashing firmware v1.0.2.0 on Minino

### ISSUE THREAD

**Issue by roozea** [ElectronicCats/Minino#3]
# Issue with flashing firmware v1.0.2.0 on Minino

## Problem Description
When flashing the v1.0.2.0 firmware using the `build_files.zip` file, the device does not start correctly. There is no animation on the screen, no sound produced, and the flash LED does not blink after resetting the device.

However, when using the `test_build_files.zip` file from the same version, the device behaves differently.

## Steps to Reproduce
1. Downloaded `build_files.zip` and `test_build_files.zip` files from version v1.0.2.0.
2. Followed the flashing instructions from the [official Minino wiki](https://github.com/ElectronicCats/Minino/wiki/2.-Get-Started#how-to-update-minino-firmware).
3. Used the following command for flashing:
   ```
   python3 -m esptool --chip esp32c6 -p /dev/cu.usbmodem101 -b 460800 --before=default_reset --after=hard_reset write_flash --flash_mode dio --flash_freq 80m --flash_size 8MB 0x0 bootloader/bootloader.bin 0x10000 minino.bin 0x8000 partition_table/partition-table.bin
   ```
4. Reset the device after flashing.

## Expected Behavior
The device should start correctly and display the normal user interface after flashing with `build_files.zip`.

## Actual Behavior
- With `build_files.zip`: No visible or audible response from the device.
- With `test_build_files.zip`: The device starts and performs the LED test, sound test, button test, and microSD test. After these tests, it displays a screen with the following information:
  - Signal: None
  - Date: 2025/78/24
  - Time: 49:67:52
  The device then remains on this signal screen, continuously refreshing the displayed information.

## Environment
- Firmware version: v1.0.2.0
- Operating system: MacOS 15
- Flashing tool: esptool (installed via brew)

## Additional Information
Is there any additional step needed for the firmware from `build_files.zip` to work correctly? Could there be an issue with the files in `build_files.zip`?

**Comment by Eric286** [comment]
Hello @roozea 

Thank you very much for letting us know this, we have carried out some tests and I recommend you download the latest published version, which is version 1.0.2.1.

Likewise, if you have the same or a different problem, please let me know.

https://github.com/ElectronicCats/Minino/releases

Have a nice Day!

**Comment by roozea** [comment]
Thank you for your assistance with this issue. I'm pleased to report that the problem has been resolved with a new firmware version provided.

After flashing the new firmware, the device is now functioning correctly. It initializes properly, entering normal operation mode as expected.

I appreciate the prompt attention and resolution to this issue. 

Given that this issue is now resolved, I'll be closing it. Thank you again for your support.
